src/server/cluster/outgoing_slot_migration.cc

```diff
@@ -57,11 +57,16 @@ class OutgoingMigration::SliceSlotMigration : private ProtocolClient {
       return;
     }
 
+    if (cancelled_) {
+      return;
+    }
+
     streamer_.Start(Sock());
```

Comment:
it can't be true, because we work with this code in one thread, and we can not cancel it before start

Reply:
It can happen - when we add the outgoing migration to the list, and then yield in this function (like when connecting, etc) and in another fiber it is cancelled.
This is actually reproducible in the cancel test after ~10 attempts :)

Reply:
I see. Could you add a comment here?